### PR TITLE
Bar Fights 2

### PR DESCRIPTION
**Pull request by KazeEspada**
Enhances barfights by making chairs smash against people. If they are weak enough, the act of smashing will knock them on their ass.

Also some backend stuff to make chairs more readable.

:cl: KazeEspada
rscadd: You can now smash chairs against people! If they are weak enough, they will knock the person over.
/:cl:

### CODE REVIEW

**Comment by Zilenan91** [comment]
So for chairs would a break chance of 5 = a 50% chance of breaking per hit or what

**Comment by KazeEspada** [comment]
break chance of 5 = 5%

the stun is supposed to be a disarm, or setup for a wrestling maneuver.

**Comment by Zilenan91** [comment]
If you want the chance for it to break to be that low then why add this at all. It's too infrequent to ever count on or ever use reliably.

**Comment by KazeEspada** [comment]
Wooden chairs is why. They are planned.

**Comment by Zilenan91** [comment]
There doesn't look to be anything here referencing wooden chair break chance.

If there is, why not just expand that functionality to regular metal chairs. Chairs are really kinda shit anyways.

**Comment by Aloraydrel** [comment]
wew

**Comment by KazeEspada** [comment]
Wooden chairs require more work and may come in this PR or a later PR. ICONS IS WHY.

**Comment by KazeEspada** [comment]
One sec, travis fucked.

**Comment by Zilenan91** [comment]
Ooh I thought you meant planned as in a guaranteed break chance not as future content. My mistake.

**Comment by Ultimate-Chimera** [comment]
Cool idea.

**Comment by KazeEspada** [comment]
WOODEN CHAIRS!

File: code/game/objects/structures/beds_chairs/chair.dm [hunk 1]
```diff
@@ -241,8 +236,57 @@
...
+
+/obj/item/chair/afterattack(atom/target, mob/living/carbon/user, proximity)
+	..()
+	if(!proximity)
+		return
+	if(prob(break_chance))
+		user.visible_message("<span class='danger'>[user] smashes \the [src] to pieces against \the [target]</span>")
+		if(iscarbon(target))
+			var/mob/living/carbon/C = target
+			if(C.health < (C.maxHealth*0.5))
```
Comment (ChangelingRain) [inline code/game/objects/structures/beds_chairs/chair.dm]: You don't need to do (C.maxHealth*0.5) here

Comment (KazeEspada) [inline code/game/objects/structures/beds_chairs/chair.dm]: That's so it use's half maxHealth?

Comment (ChangelingRain) [inline code/game/objects/structures/beds_chairs/chair.dm]: They're entirely unneeded for this, you should only use parentheses if you need byond to do operations in a specific order, and you don't need that here.

Comment (KazeEspada) [inline code/game/objects/structures/beds_chairs/chair.dm]: You want me to make a commit to remove two parentheses?

**Pull request merged**